### PR TITLE
feat: add signinPath to SdkConfig

### PR DESCRIPTION
**Pull request by xiaocode**
Fix: https://github.com/casdoor/casdoor-js-sdk/issues/34

### CODE REVIEW

**Comment by hsluoyz** [comment]
@xiaocode use the same committer as your github account, and sign CLA

image

File: src/sdk.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export interface SdkConfig {
     appName: string, // the name of your Casdoor application, e.g., "app-casnode"
     organizationName: string // the name of the Casdoor organization connected with your Casdoor application, e.g., "casbin"
     redirectPath?: string // the path of the redirect URL for your Casdoor application, will be "/callback" if not provided
+    signinPath?: string // the path of the signin URL for your Casdoor applcation, will be "/api/signin" if not provided
```
Comment (hsluoyz) [inline src/sdk.ts:21]: Also add the new usage to README.md

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: src/sdk.ts (hunk 1 above)
Comment (xiaocode) [reply src/sdk.ts:21]: I have done that

**Comment by hsluoyz** [comment]
@xiaocode your committer Email should be added to your GitHub account:

image

**Review by hsluoyz — APPROVED** [review]

**Comment by xiaocode** [comment]
> @xiaocode your committer Email should be added to your GitHub account:
> 
> Also refer to: https://casdoor.org/docs/contributing#modify-prs
> 
> image

Done!

**Pull request merged**